src/main/java/FTPServer.java

```diff
@@ -294,10 +295,13 @@ private void sendDataBlock(byte[] data) throws IOException {
 
         // Send each data block sequentially
         for(DataMessage m : messages) {
+        		LOG.logVerbose("Sending to Client " + packet.getSocketAddress());
```

Comment:
I'll need to remove a bunch of this logging, I was justing using it to debug the problem. However, I think we should add a lot more detail in the verbose mode eventually.